src/pipeline.ml

```diff
@@ -339,6 +424,20 @@ let tarides ?app ?notify:channel ?filter ~sched ~staging_auth () =
   (* GitHub organisations to monitor. *)
   let ocurrent = Build.org ?app ~account:"ocurrent" 12497518 in
   let ocaml_bench = Build.org ?app ~account:"ocaml-bench" 19839896 in
+  let tarides = Build.org ?app ~account:"tarides" 21197588 in
+
+  let build_with_context (org, name, builds) =
+    Docker_context_build.repo ?channel ~web_ui ~org ~name builds (* XXX: verify with the unikernel version. *)
```

Comment:
Is there something that needs doing for this comment?

Reply:
No, I can remove it. It is not relevant anymore...